CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Cromwell Change Log
 
+## 60 Release notes
+
+### Warning! Breaking Change: Carboniting Removed
+
+All carboniting functionality has been removed from Cromwell. If you had previously enabled carboniting and deletion,
+any workflows marked as ArchivedAndPurged in your database will no longer be accessible via the cromwell metadata API.
+This is in line with future archiving plans for Cromwell which will allow a more scalable form of archiving, but not
+automated retrieval via the service APIs.
```

Comment:
Re-ordered so that people to whom this doesn't apply are more likely to (appropriately) skim past and not freak out at "breaking change!"

Also added some flavor to help answer "OMG! They removed carboniting from Cromwell, do we use that!?"

"No replacement is planned" may be a bit controversial but I'm referring to no _documented_ replacements, I don't think we should add the new archiving to our publicly supported featureset (Terra only)
```suggestion
### Hybrid metadata storage ("carboniting") removed

Breaking change only for customers who explicitly enabled `carbonite-metadata-service` in their configuration to split metadata storage between a relational database and Google Cloud Storage.

Carboniting functionality has been removed from Cromwell. No replacement is planned.

If you had previously enabled carboniting and deletion, any workflows marked as `ArchivedAndPurged` in your database will no longer be accessible via the Cromwell metadata API.

There is no effect for customers who store metadata in a relational database only (most common).
```

Reply:
I re-imagined the phrasing a little bit. Let me know what you think. 

I also opted to omit the "no replacement" line, because it seemed superfluous for the 99%, and just a little too strong for my personal pedantry preferences.